src/components/TermsAndConditions.js

```diff
@@ -35,6 +40,13 @@ class TermsAndConditionsComponent extends Component {
       // Scroll up -- otherwise we're going to a tiles page that may also be scrolled down
       document.body.scrollTop = document.documentElement.scrollTop = 0;
       this.props.termsSigned();
+
+      if (
+        this.props.clientConfiguration.termsAndConditionsSetAgreedCookie ===
```

Comment:
only set cookie if field in client_configuration "termsAndConditionsSetAgreedCookie" is true